diff --git a/fourth.c b/fourth.c
index ef4a85a..ec84a4c 100644
--- a/fourth.c
+++ b/fourth.c
@@ -1,4 +1,5 @@
 #include<stdio.h>
 int main(){
 	printf("I am adding this on main branch");
+	//updated for patch
 }
diff --git a/third.c b/third.c
index e28482f..7e34195 100644
--- a/third.c
+++ b/third.c
@@ -1,4 +1,6 @@
 #include<stdio.h>
 int main(){
 	printf("third.c");
+	//adding to create patch
+	printf("Creating changes for patch");
 }
